api/File.json

```diff
@@ -0,0 +1,307 @@
+{
+  "api": {
+    "File": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/File",
+        "support": {
+          "webview_android": {
+            "version_added": false
+          },
+          "chrome": {
+            "version_added": "13"
+          },
+          "chrome_android": {
+            "version_added": null
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": [
+            {
+              "version_added": "7"
+            },
+            {
+              "version_added": "3",
+              "version_removed": "7",
+              "notes": "Non-standard implementation."
+            }
+          ],
+          "firefox_android": [
+            {
+              "version_added": "7"
+            },
+            {
+              "version_added": "4",
+              "version_removed": "7",
+              "notes": "Non-standard implementation."
+            }
+          ],
+          "ie": {
+            "version_added": "10"
+          },
+          "opera": {
+            "version_added": "11.5"
+          },
+          "opera_android": {
+            "version_added": "11.1"
+          },
+          "safari": {
+            "version_added": "6"
+          },
+          "safari_ios": {
+            "version_added": "6"
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "File": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/File/File",
+          "description": "<code>File()</code> constructor",
+          "support": {
+            "webview_android": {
+              "version_added": "4.4.4"
```

Comment:
Change to `true`. Webview has parity with Chrome versions, not Android versions.